src/main_nep/nep4.cu

```diff
@@ -413,8 +455,7 @@ void NEP4::find_force(
   Parameters& para, const float* parameters, Dataset& dataset, bool calculate_q_scaler)
```

Comment:
If the descriptor has already by normalized before applying GNN, it does not need to be normalized again here.

Reply:
I mean you can remove the argument `g_q_scaler` for the `apply_ann` function.

Reply:
Can I really do that? Don't I need to keep q_scalar for line 303? There it is used to scale the forces as well.

```python
    for (int d = 0; d < annmb.dim; ++d) {
      g_Fp[n1 + d * N] = Fp[d] * g_q_scaler[d];
    }
```

Reply:
yes, you can remove the use of `g_q_scaler` here. Later when you calculate the derivatives, you  can take the noralization into account at a different place.